gotocompany/compass/v1beta1/service.proto

```diff
@@ -346,6 +346,20 @@ service CompassService {
     };
   }
 
+  rpc DeleteAssets(DeleteAssetsRequest) returns (DeleteAssetsResponse) {
```

Comment:
I am still unsure whether having DELETE with query params is a good idea or not.
Or should we just use POST with req body for this.

Reply:
After talking again, we decided to send a POST request with a body, following deletion API in elastic search https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-delete-by-query.html

Thank you mas 🙏